<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InventoryRemindItemWgt</class>
 <widget class="QWidget" name="InventoryRemindItemWgt">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>280</width>
    <height>38</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>InventoryRemindItemWgt</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QLabel{
font-family:&quot;微软雅黑&quot;;
font-size:14px;
}
#lbl_name{
color:rgba(0,0,0,0.9);
}
#lbl_time{
color:rgba(0,0,0,0.3);
}</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLabel" name="lbl_logo">
     <property name="minimumSize">
      <size>
       <width>14</width>
       <height>14</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>14</width>
       <height>14</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="lbl_name">
     <property name="text">
      <string>进货提醒</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="lbl_time">
     <property name="text">
      <string>14:34:23</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
